static/app/views/issueDetails/traceTimeline/traceTimeline.tsx

```diff
@@ -1,31 +1,44 @@
 import {useRef} from 'react';
 import styled from '@emotion/styled';
 
+import Feature from 'sentry/components/acl/feature';
 import ErrorBoundary from 'sentry/components/errorBoundary';
 import QuestionTooltip from 'sentry/components/questionTooltip';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import type {Event} from 'sentry/types/event';
 import useRouteAnalyticsParams from 'sentry/utils/routeAnalytics/useRouteAnalyticsParams';
 import {useDimensions} from 'sentry/utils/useDimensions';
+import useOrganization from 'sentry/utils/useOrganization';
 
 import {TraceTimelineEvents} from './traceTimelineEvents';
-import {useTraceTimelineEvents} from './useTraceTimelineEvents';
+import {EventItem} from './traceTimelineTooltip';
+import {type TimelineEvent, useTraceTimelineEvents} from './useTraceTimelineEvents';
+
+const twoIssues = 2;
```

Comment:
nit: could use a more descriptive name, and uppercase naming because it's a constant.